### PR TITLE
Fix some styling + ux issues with the annotation creation component

### LINKED ISSUE #39: only first line of annotation text-area is clickable
* clicking on the first line of the text-area correctly places the cursor
* however, if the click is on the second line or lower, the cursor is not placed

---

observed on Firefox 62

### LINKED ISSUE #54: Add margin to left edge of New/Edit annotation panel
I know the New/Edit annotation panel is only in a first draft version (there are more features proposed in #4) but we could make it feel a little more polished and friendly to users with a minor styling update.

Currently there is no whitespace between the left edge of the panel and the left-edge of the panel title and panel content elements:

Screen Shot 2021-05-12 at 9 39 52 AM

I think we're using `padding-left: 16px;` for the left edge margin on the default window sidebar; I suggest using the same value for the New/Edit annotation panel.

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #39 
Fixes #54

### CODE REVIEW

**Comment by ggeisler** [comment]
Looks good to me. 👍

File: src/TextEditor.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ class TextEditor extends Component {
     this.onChange = this.onChange.bind(this);
     this.handleKeyCommand = this.handleKeyCommand.bind(this);
     this.handleFormating = this.handleFormating.bind(this);
+    this.handleFocus = this.handleFocus.bind(this);
+    this.editorRef = React.createRef();
+  }
+
+  /**
+   * This is a kinda silly hack (but apparently recommended approach) to
+   * making sure the whole visible editor area, not just the first line.
```
Comment (mjgiarlo) [inline src/TextEditor.js:29]: This comments reads a bit awkward to my 👀. Is it missing a word?

**Review by mjgiarlo — APPROVED** [review]

File: src/TextEditor.js (hunk 1 above)
Comment (mjgiarlo) [reply src/TextEditor.js:29]: Consider changing and then merge?

Comment (cbeer) [reply src/TextEditor.js:29]:
```suggestion
   * making sure the whole visible editor area is clickable, not just the first line.
```

**Review by camillevilla — APPROVED** [review]

**Pull request merged**